src/routes/spent/index.ts

```diff
@@ -0,0 +1,10 @@
+import ExpenseController from '../spent/controller/ExpenseController'
+import {Router} from 'express'
+
+const spentRouter = Router()
+
+const expense = new ExpenseController()
```

Comment:
Acho que dessa forma fica um pouco mais explícito o conteúdo da variável:

```suggestion
const expenseController = new ExpenseController()
```